src/YunInternet/Libvirt/Configuration/Domain/Device/Disk.php

```diff
@@ -9,6 +9,7 @@
 
 
 use YunInternet\Libvirt\Configuration\Domain\Device\Disk\Address;
+use YunInternet\Libvirt\Configuration\Domain\Device\Disk\backingStore;
```

Comment:
Hi, the name of this class should still be "BackingStore". Only the method name should be "backingStore". As you can see, the method name of address is "address", but its class name is "Address".

Example: 
```
...
use YunInternet\Libvirt\Configuration\Domain\Device\Disk\BackingStore;
...
 * @method BackingStore backingStore()
...
class Disk ... {
    ...
    protected $singletonChildWrappers = [
        ...
        "backingStore" => BackingStore::class,
    ];
    ...
}
```
Then you can get your BackingStore instance by invoking $disk->backingStore().